src/components/subscriptions/LicenseAllocationNavigation.jsx

```diff
@@ -50,7 +52,7 @@ export default function LicenseAllocationNavigation() {
                 'btn btn-link btn-block pl-0 text-left',
                 { 'font-weight-bold': activeTab === tab.key },
               )}
-              onClick={() => setActiveTab(tab.key)}
+              onClick={() => { setActiveTab(tab.key); setCurrentPage(DEFAULT_PAGE); }}
```

Comment:
This has been done so that when user changes the tab then he will be shown the records of the specific tab from page no1.

Reply:
Might recommend creating a new function that does both `setActiveTab` and `setCurrentPage`, and using that function here (to avoid putting 2 function calls in a single callback).